object/id/id.go

```diff
@@ -141,7 +141,7 @@ func (id *ID) Unmarshal(data []byte) error {
 		return err
 	}
 
-	copy(id[:], v2.GetValue())
+	*id = ID(v2.GetValue())
```

Comment:
also can panic if `v2.GetValue` is too short

Reply:
one more bug revealer, will fix

Reply:
fixed